contrib/docs/tutorials/authenticate-api-requests.md

```diff
@@ -160,16 +160,15 @@ const app = createApp({
           providers={['guest', 'custom', ...providers]}
           title="Select a sign-in method"
           align="center"
-          onResult={async result => {
-            // When logged in, set a token cookie
-            if (typeof result.getIdToken !== 'undefined') {
+          onSignInSuccess={async (identityApi: IdentityApi) => {
+            if (typeof identityApi?.getIdToken !== 'undefined') {
```

Comment:
This check isn't really relevant anymore so you can remove the `if` statement. `setTokenCookie` should be called always, and changed to accept an entire `IdentityApi` instance instead of the token getter. Then its code should be updated to do

```ts
const { token } = await identityApi.getCredentials();
```

and be prepared to deal with that token might end up being undefined; if it is, it can just try again 10 seconds later or whatever.